recipe/meta.yaml

```
@@ -44,18 +45,19 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=8,<9
+    - arrow-cpp >=10,<11
     - pyarrow
     - numpy
     - setuptools >=40.6.0
     - wheel
   run:
     - python
+    - packaging
     - pyarrow
     - {{ pin_compatible('arrow-cpp', max_pin='x') }}
     - asn1crypto >0.24.0,<2.0.0
     - cffi >=1.9,<2.0.0
-    - cryptography >=3.1.0,<37.0.0
+    - cryptography >=3.1.0,<39.0.0
```

```suggestion
    - cryptography >=3.1.0,<41.0.0
```
https://github.com/snowflakedb/snowflake-connector-python/blob/v3.0.3/setup.cfg#L48

---

I was following the conda-forge recipe, but I guess we should follow the upstream instead?

---

yes, often cf only update the version of sha and upload if it builds